frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.tsx

```diff
@@ -520,6 +522,40 @@ export const WhatsNewMenu = (props: Props) => {
     entries.push(firefoxIntegrationAnnouncement);
   }
 
+  const mailingListAnnouncement: WhatsNewEntry = {
+    title: l10n.getString("whatsnew-feature-mailing-list-heading"),
+    snippet: l10n.getString("whatsnew-feature-mailing-list-snippet"),
+    content: (
+      <WhatsNewContent
+        description={l10n.getString(
+          "whatsnew-feature-mailing-list-description"
+        )}
+        heading={l10n.getString("whatsnew-feature-mailing-list-heading")}
+        image={MailingListHero}
+        cta={
+          <a
+            className={styles.cta}
+            href="https://www.mozilla.org/en-US/newsletter/security-and-privacy/"
```

Comment:
question: should this have some GA/UTM params?

Reply:
And maybe strip the locale code, so it can auto-detect the right locale :)

```suggestion
            href="https://www.mozilla.org/newsletter/security-and-privacy/"
```